### ISSUE TITLE
Authentication failed to push origin

### ISSUE THREAD

**Issue by YudiGuzman** [desktop/desktop#17486]
### The problem

When I try to push the origin of one of my repositories, the desktop GitHub tells me "Authentication failed" because they are unable to authenticate with ``github.com``

I have tried to enter many times my username and password (Including the GitHub web's username and password), but GitHub Desktpo thorught out the same authentication error.

Furthermore, I was trying to change the password but did not find any way to do it.

Thanks for your support.

### Release version

Version 3.3.3 (x64)

### Operating system

Windows 11

### Steps to reproduce the behavior

1. Go to Push commits to the origin remote because I 1 local commit waiting to be pushed to the remote
2. Click on Push origin botton
3. See error 'Authentication failed' box
![GitHub Desktop_Authentication failed](https://github.com/desktop/desktop/assets/65938408/8786d7ba-49ff-4dd4-9d17-7cc42f2fb3cb)

### Log files

[2023-08-23.desktop.production.log](https://github.com/desktop/desktop/files/12805642/2023-08-23.desktop.production.log)
[2023-08-24.desktop.production.log](https://github.com/desktop/desktop/files/12805643/2023-08-24.desktop.production.log)
[2023-10-04.desktop.production.log](https://github.com/desktop/desktop/files/12805644/2023-10-04.desktop.production.log)

### Additional context

Please explain how to manage the GitHub Web and GitHub Desktop credentials because I am confused about this issue.

**Comment by steveward** [comment]
@YudiGuzman can you go to `File` > `Options` > `Accounts` and verify you are logged in to there?

**Comment by YudiGuzman** [comment]
@steveward thanks for your prompt reply. I was signed out. When I logged in to the GitHub Browser, I got to push my repository. However, I am not clear with the GitHub Web and GitHub Desktop credentials. I mean I understand I need only the credentials GitHub Browser. Thanks for your help.
![GitHub Desktop_Signed out GitHub Browser](https://github.com/desktop/desktop/assets/65938408/10094357-33fe-4938-9ce0-f8d3153e1b7d)
